torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1,3 +1,4 @@
+""""""
```

Comment:
What's up with this series of quotes?

Reply:
Oh, this got committed! I'm so sorry, will remove this.

Reply:
Fixed. Thanks!